spec/models/group_spec.rb

```diff
@@ -0,0 +1,6 @@
+require 'rails_helper'
+
```

Comment:
- [ ] When I tried to run the tests, I got the following errors. Pleas efix them.

![image](https://user-images.githubusercontent.com/62290545/231914945-8857e509-780c-47d7-8f91-c6f8e8e508a2.png)

- [ ] Kindly make sure to implement unit and integration tests for all the components, as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=Create%20unit%20and%20integration%20tests%20for%20all%20the%20most%20important%20components%20of%20your%20RoR%20application.) requirement.

Reply:
:warning: Use this to run integration test `rspec test/integration`

Reply:
well resolved! ✔️